packages/docs-site/src/content/docs/docs/guides/schema.mdx

```diff
@@ -18,7 +18,19 @@ You'll always have a `.dmno/config.mts` file at the root of your project, and if
 A typical DMNO project structure:
 
 <Tabs syncKey='repoType'>
-  <TabItem label="Monorepo">
+  <TabItem label="Single repo">
+    <FileTree>
+        - / (root of your project)
+          - .dmno/ (your workspace config)
+            - **config.mts**
```

Comment:
maybe add all the files on this one, and then on the monorepo case, just include the .dmno folder itself?

Reply:
yeah good call